<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <node name="base_node" pkg="b2_base" type="base_node.py">
        <param name="dev_name" type="str" value="/dev/ttyACM0"/>

    </node>
    <node name="roboclaw" pkg="roboclaw_driver" type="roboclaw_node.py">
        <param name="dev_name" type="str" value="/dev/ttyACM0"/>
        <param name="baud" type="int" value="115200"/>
        <param name="address" type="int" value="128"/>
        <param name="loop_hz" type="int" value="10"/>
        <param name="deadman_secs" type="int" value="10"/>
        <param name="test_mode" type="bool" value="false"/>
    </node>
    <test test-name="roboclaw_node_nodetest" pkg="roboclaw_driver" type="test_roboclaw_node.py"/>
</launch>